app/models/snap_application_attributes.rb

```diff
@@ -221,7 +220,7 @@ def financial_accounts(*types)
       end
     end
 
-    included.include?(true)
+    bool_to_checkbox(included.include?(true))
```

Comment:
`included.include?(true)`

Could this be pulled out to something that describes it a little better? 😸

Reply:
done at 5fcad41 ;) -- agree that the code situation was funky before 💩